.changeset/orange-files-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@ceramicnetwork/streamid": patch
+---
+
+chore(streamid): Stricter code. Also, now NoThrow functions really do not throw an error.
```

Comment:
I think @smrz2001 reverted the use of changesets - can we take this file out for now?

Reply:
removed.